modules

```diff
@@ -2,7 +2,3 @@ GLUON_SITE_FEEDS='ffffm_packages ssidchanger'
 
 PACKAGES_FFFFM_PACKAGES_REPO=git://github.com/freifunk-ffm/packages.git
 PACKAGES_FFFFM_PACKAGES_COMMIT=94406af836d50b3d0435565953b9a89cfcc69e73
-
-PACKAGES_SSIDCHANGER_REPO=https://github.com/freifunk-nord/gluon-ssid-changer.git
```

Comment:
Should be "discussed" on a wider level? Maybe reinclude when we have BATMAN 15 and the feature is upstream (https://github.com/freifunk-gluon/gluon/pull/1649)

Reply:
We should discuss this again when this feature is upstream

Reply:
Please discussed this 'removing' with the big mesh maintainer. It looks like the package benefit is not understood.